### ISSUE TITLE
Can ditilling task be trained for very large steps?

### ISSUE THREAD

**Issue by splinter21** [hao-ai-lab/FastVideo#129]
when steps>1200, loss: gg
![321](https://github.com/user-attachments/assets/e84a1eba-3fa3-435a-8494-ed682883d3c2)

the training dataset is fastvideo default wukong dataset

**Comment by rlsu9** [comment]
Thanks for your feedback!
Please don't use default wukong dataset for distill job, the default one is to give a sample for the format of finetune data, to carry out the distill, we recommend to use the `FastVideo/HD-Mixkit-Finetune-Hunyuan` dataset from huggingface.

**Comment by GFENGG** [comment]
what is the suitable training steps and batch size when distilling hunyuanvideo model?

**Comment by jzhang38** [comment]
https://github.com/hao-ai-lab/FastVideo/blob/dd75ee8509943f2872b1b196d245afe1961c629b/scripts/distill/distill_hunyuan.sh#L7

You should be able to reproduce FastHunyuan with this script.

**Comment by GFENGG** [comment]
Hello,  can i use  image-video joint training in the distilling process? 
and image-video joint training or using only video datasets, Which is better?

> https://github.com/hao-ai-lab/FastVideo/blob/dd75ee8509943f2872b1b196d245afe1961c629b/scripts/distill/distill_hunyuan.sh#L7
> 
> You should be able to reproduce FastHunyuan with this script.

**Comment by rlsu9** [comment]
@GFENGG No, the image-video joint training is an alternate choice for finetune, for distilling process, please use video datasets. For your reference, FastHunyuan is distilled on `FastVideo/HD-Mixkit-Finetune-Hunyuan`.
